Makefile

```
@@ -0,0 +1,30 @@
+FILES=main.js manifest.json
+
+clean:
+	-rm -rf *.js *.css
+
+dev:
+	npm run dev
+
+build:
+	npm run build
+
+install: install-user-vault
```

checkmake (0.3.2)

[warning] Target "install" should be declared PHONY.

(phonydeclared)